packages/base-styles/_variables.scss

```diff
@@ -71,7 +71,7 @@ $shadow-modal: 0 10px 10px rgba($black, 0.25);
  * Editor widths.
  */
 
-$sidebar-width: 280px;
+$sidebar-width: 280px + 17px; // 17px is the maximum width of a scrollbar across all browsers, this ensures that the content area is at least 280px wide
```

Comment:
Even this little change affects a large portion of Gutenberg and we'd need some extensive testing to ensure it doesn't break anything. It's used for more than just width, so it can theoretically break things. I believe it deserves its own PR and its own separate discussion.

Reply:
Ah ! Indeed, didn't see all the usage of this variables. It can be reverted to just 280px but the sidebar will then have a smaller inner size because of the sidebar